pkg/operator/operator_test.go

```diff
@@ -574,8 +582,14 @@ func TestMAOConfigFromInfrastructure(t *testing.T) {
 				proxy := tc.proxy.DeepCopy()
 				objects = append(objects, proxy)
 			}
-			stopCh := make(<-chan struct{})
-			optr := newFakeOperator(nil, objects, stopCh)
+
+			imagesJSONFile, err := createImagesJSONFromManifest()
```

Comment:
Is it possible to create this file once during test suite setup?

Reply:
Well not test *suite*, but certainly once for this test.

Reply:
I've moved this out of the loop in the couple of places where it's relevant.